applications/crossbar/src/crossbar.hrl

```diff
@@ -121,6 +121,7 @@
                          ,'cb_token_auth'
                          ,'cb_token_restrictions'
                          ,'cb_transactions'
+                         ,'cb_tray'
```

Comment:
I don't think this should be added to the default list. It should be explicitly enabled by sysadmins, including new installs.

Reply:
Ok that makes sense, I will remove this.